style.css

```diff
@@ -0,0 +1,382 @@
+@import url('https://fonts.cdnfonts.com/css/cocogoose');
+
+* {
+  font-family: 'COCOGOOSE', sans-serif;
+}
+
+body {
+  width: 100%;
+}
+
+.main-nav,
+.navigation,
+.link-to-program,
+.last-text {
+  display: none;
+}
+
+#first-header {
+  background-color: rgba(227, 224, 224, 0.9);
+  height: 10vh;
+}
+
+.top-bg {
+  background-image: url("assets/backgroundImg.png");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+#menu-image {
+  font-size: 30px;
+}
+
+.hero {
+  color: rgb(255, 64, 0);
+}
+
+.hero-first-title {
+  font-size: 40px;
+  font-weight: 700;
+}
+
+.message-p {
+  background-color: rgb(194, 194, 194, 0.4);
+  font-weight: 600;
+}
+
+.text-time {
+  font-weight: 700;
+  font-size: 30px;
+}
+
+.section-title-white {
+  color: white;
+}
+
+.section-title-black {
+  color: black;
+}
+
+.line-name {
+  border: 1px solid rgb(255, 64, 0);
+  width: 40px;
+}
+
+.program {
+  background-image: url("assets/violin.png");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.program-card {
+  display: flex;
+  flex-direction: row;
+  background-color: rgb(90, 90, 90, 0.7);
+}
+
+.program-title {
+  color: rgb(255, 64, 0);
+  font-weight: 500;
+  font-size: 20px;
+}
+
+#cc-btn {
+  background-color: #fff;
+  color: rgb(255, 64, 0);
+  border: 4px solid aquamarine;
+  text-decoration: none;
+  margin-top: 20px;
+}
+
+#cc-btn:hover {
+  background-color: rgb(255, 64, 0);
```

Comment:
- [ ] As suggested already, I think you should use the color in the Behance design, the color used in the design is `#EC5242`  instead of `rgb(255, 64, 0)`:pray: